### PR TITLE
flashback: add alert msg in flashback statement

### PR DESCRIPTION
**Pull request by Defined2014**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add a warning when using FLASHBACK.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ summary: TiDB 数据库中 FLASHBACK CLUSTER TO TIMESTAMP 的使用概况。
 
 TiDB v6.4.0 引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
 
+> **警告：**
+>
+> 在 v7.1.0 版本中使用该功能可能会出现 FLASHBACK 完成后部分 region 无法恢复的问题，请尽量避免在 v7.1.0 中使用该功能，相关 [issue](https://github.com/pingcap/tidb/issues/44292)。
```
Comment (JmPotato) [inline sql-statements/sql-statement-flashback-to-timestamp.md:12]:
```suggestion
> 在 v7.1.0 版本中使用该功能可能会出现 FLASHBACK 完成后部分 Region 无法恢复的问题，请尽量避免在 v7.1.0 中使用该功能，相关 [issue](https://github.com/pingcap/tidb/issues/44292)。
```

**Review by JmPotato — APPROVED** [review]

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ summary: TiDB 数据库中 FLASHBACK CLUSTER TO TIMESTAMP 的使用概况。
 
 TiDB v6.4.0 引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
 
+> **警告：**
+>
+> 在 v7.1.0 版本中使用该功能可能会出现 FLASHBACK 完成后部分 Region 无法恢复的问题，请尽量避免在 v7.1.0 中使用该功能，相关 [issue](https://github.com/pingcap/tidb/issues/44292)。
+>
+> 如已出现该问题，可以使用 [BR 快照备份与恢复](/br/br-snapshot-guide.md#tidb-快照备份与恢复使用指南)功能进行数据恢复。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-flashback-to-timestamp.md:14]:
```suggestion
> **警告：**
>
> 在 TiDB v7.1.0 中使用该功能可能会出现 FLASHBACK 完成后部分 Region 无法恢复的问题。请尽量避免在 v7.1.0 中使用该功能。详情可见此 [issue](https://github.com/pingcap/tidb/issues/44292)。
>
> 如果已经出现该问题，可以使用 [TiDB 快照备份与恢复](/br/br-snapshot-guide.md#tidb-快照备份与恢复使用指南)功能进行数据恢复。
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 3]
```diff
@@ -7,6 +7,12 @@ summary: TiDB 数据库中 FLASHBACK CLUSTER TO TIMESTAMP 的使用概况。
 
 TiDB v6.4.0 引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
 
+> **警告：**
+>
+> 在 TiDB v7.1.0 中使用该功能可能会出现 FLASHBACK 完成后部分 Region 无法恢复的问题。请尽量避免在 v7.1.0 中使用该功能。详情可见此 [issue](https://github.com/pingcap/tidb/issues/44292)。
+>
+> 如果已经出现该问题，可以使用 [TiDB 快照备份与恢复](/br/br-snapshot-guide.md#tidb-快照备份与恢复使用指南)功能进行数据恢复。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-flashback-to-timestamp.md:14]:
```suggestion
> 如果已经出现该问题，可以使用 [TiDB 快照备份与恢复](/br/br-snapshot-guide.md)功能进行数据恢复。
```

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ summary: TiDB 数据库中 FLASHBACK CLUSTER TO TIMESTAMP 的使用概况。
 
 TiDB v6.4.0 引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
 
+> **警告：**
+>
+> 在 TiDB v7.1.0 中使用该功能可能会出现 FLASHBACK 完成后部分 Region 无法恢复的问题。请尽量避免在 v7.1.0 中使用该功能。详情可见此 [issue](https://github.com/pingcap/tidb/issues/44292)。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-flashback-to-timestamp.md:12]:
```suggestion
> 在 TiDB v7.1.0 中使用该功能可能会出现 FLASHBACK 完成后部分 Region 无法恢复的问题。请尽量避免在 v7.1.0 中使用该功能。详情可见 [#44292](https://github.com/pingcap/tidb/issues/44292)。
```

**Review by qiancai — APPROVED** [review]

**Review by bb7133 — APPROVED** [review]
LGTM

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 5]
```diff
@@ -7,6 +7,12 @@ summary: TiDB 数据库中 FLASHBACK CLUSTER TO TIMESTAMP 的使用概况。
 
 TiDB v6.4.0 引入了 `FLASHBACK CLUSTER TO TIMESTAMP` 语法，其功能是将集群的数据恢复到特定的时间点。
 
+> **警告：**
+>
+> 在 TiDB v7.1.0 中使用该功能可能会出现 FLASHBACK 完成后部分 Region 无法恢复的问题。请尽量避免在 v7.1.0 中使用该功能。详情可见 [#44292](https://github.com/pingcap/tidb/issues/44292)。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-flashback-to-timestamp.md:12]:
```suggestion
> 在 TiDB v7.1.0 中使用该功能可能会出现 FLASHBACK 完成后部分 Region 仍处于 FLASHBACK 过程中的问题。请尽量避免在 v7.1.0 中使用该功能。详情可见 [#44292](https://github.com/pingcap/tidb/issues/44292)。
```
@Defined2014 PTAL

**Pull request merged**